_posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "DSAA Wrapped"
+description: |
+  Reflecting on 2022.
+author:
+  - name: Chloe Pou-Prom, Meggie Debnath, Derek Beaton
+date: 2022-12-13
+output:
+  distill::distill_article:
+    self_contained: false
+draft: true
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+The end of the year is a time for reflection, reminiscing, and sharing screenshots of our Spotify Wrapped. For those who are unfamiliar, Spotify Wrapped is a yearly marketing campaign by Spotify, where they compile streaming data and create a personalized graphic for each user. ^[We surveyed the members on DSAA and found that the most listened music genre is: ???] 
```

Comment:
Could be fun to ask people what their top genre on Spotify Wrapped (or the Apple Music equivalent) was and we could include this here :D

Reply:
looks like the top answer was pop (and some interesting variations of indie ~ indie soul, indie pop, indie Québécois)